app/src/main/java/kr/sesac/aoao/android/friend/FriendActivity.kt

```diff
@@ -0,0 +1,72 @@
+package kr.sesac.aoao.android.friend
+
+import android.os.Bundle
+import android.text.Editable
+import android.text.TextWatcher
+import android.view.View
+import android.view.ViewGroup
+import android.widget.*
+
+import androidx.appcompat.app.AppCompatActivity
+import kr.sesac.aoao.android.R
+
+public class FriendActivity : AppCompatActivity() {
+    private lateinit var searchEditText: EditText
+    private lateinit var friendsListView: ListView
+    private lateinit var titleTextView: TextView
+    private lateinit var adapter: ArrayAdapter<String>
+
+    private val dummyData = listOf(
+        "김유빈", "김은서", "김은솔", "엄상은", "이상민", "이혜연", "황수연", "최정윤"
+    )
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_friend)
+
+        titleTextView = findViewById(R.id.titleTextView)
+        searchEditText = findViewById(R.id.searchEditText)
+        friendsListView = findViewById(R.id.friendsListView)
```

Comment:
```suggestion
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_friend)

        titleTextView = binding.titleTextView
        searchEditText = binding.searchEditText
        friendsListView = binding.friendsListView
```

view binding 을 사용하시면 뷰 객체에 더 간단하게 접근할 수 있습니다!! view binding 을 사용하는 과정은

```text
레이아웃 정의 -> 빌드(망치모양 아이콘) -> binding 선언
```

binding 객체의 이름은 레이아웃의 이름을 따라 가는 듯 합니다! 제가 구현한 Todo 관련 액티비티를 보시면 이해가 될 것 같습니당 ㅎㅎ